src/index.html

```diff
@@ -15,8 +15,94 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
```

Comment:
Ensure that the `src` attribute for the logo image points to the correct path. If the image is not displaying, verify the path and file name.